spec/controllers/presentations_controller_spec.rb

```diff
@@ -7,4 +7,12 @@
     it { is_expected.to render_template :new }
     it { is_expected.to be_successful }
   end
+
+  describe 'GET #show' do
+    subject { get :show, params: { id: 1 } }
```

Comment:
There is a chance (albeit small) that the presentation being created here might not have an ID of `1`. You could write this like:

```
describe "GET #show" do
  let(:presentation) { FactoryGirl.create :presentation }
  subject { get :show, params:  { id: presentation.id } }

  # stuff
end
```

Using the presentation in the subject will also allow us to removing the bang from the let.